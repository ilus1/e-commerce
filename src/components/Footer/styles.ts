import styled from 'styled-components'

import colors from '../../constants/colors'

export const Container = styled.footer`
  display: flex;
  justify-content: center;
  align-items: center;

  width: 100%;
  height: 34px;
  background-color: ${colors.footer};
  font-size: 12px;
`
