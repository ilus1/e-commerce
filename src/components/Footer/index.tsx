import { Container } from './styles'

import { footerMessage } from '../../constants/utils'

function Footer() {
  return (
    <Container>
      { footerMessage }
    </Container>
  )
}

export default Footer
