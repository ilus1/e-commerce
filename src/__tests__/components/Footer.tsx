import renderer from 'react-test-renderer'

import Footer from '../../components/Footer'

describe('Teste do componente Footer', () => {
  it('Testa carregamento correto', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON()

    expect(tree).toMatchSnapshot();
  });
});
