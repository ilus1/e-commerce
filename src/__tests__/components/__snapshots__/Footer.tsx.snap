// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste do componente Footer Testa carregamento correto 1`] = `
<footer
  className="sc-bdfCDU kdpXOj"
  data-testid="footer"
>
  OnlineStore Â© Todos os direitos reservados
</footer>
`;
